helm/sematic-server/templates/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.ingress.enabled -}}
+{{- if .Values.ingress.create -}}
```

Comment:
Do you also need to update `values.yaml` [here](https://github.com/sematic-ai/sematic/blob/main/helm/sematic-server/values.yaml#L95)?

Reply:
good catch, fixed!